includes/class-recaptcha.php

```diff
@@ -366,6 +366,10 @@ public static function verify_recaptcha_on_checkout() {
 		if ( ! $should_verify_captcha ) {
 			return;
 		}
+		// Bail if validating form.
+		if ( isset( $_POST['woocommerce_checkout_update_totals'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
Non-blocking suggestion: would it be useful to create a utility method in Blocks' `class-modal-checkout.php` to determine whether a form submission request is a modal checkout form validation request, in case we want to check other things in addition to this specific `POST` value?

Reply:
Not sure... We use `woocommerce_checkout_update_totals` specifically because that's what Woo provides. We don't have other ways to validate the checkout form.

Reply:
And that's fine, so right now such a method could simply be

```php
public static function is_validation_request() {
	return isset( $_POST['woocommerce_checkout_update_totals'] ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
}
```

But it gives us the option to add other conditions like checking our own custom params if we ever want to in the future. Definitely not a requirement at this point.

Reply:
At this point, the code is about Woo's `update_totals`. By calling a Block's method we'd be making it not about Woo, but our code. It may not benefit the codebase unless we identify other use cases.